src/ssh/azext_ssh/target_properties_utils.py

```diff
@@ -0,0 +1,203 @@
+        return compute_client.virtual_machines.get(resource_group_name, vm_name)
+    except Exception:
+        return None
+
+
+# This function is used to get the properties needed needed from an Azure Arc Server
+def _request_arc_server_properties(cmd, resource_group_name, vm_name):
+    from .aaz.latest.hybrid_compute.machine import Show as ArcServerShow
+    # pylint: disable=broad-except
+    try:
+        get_args = {'resource_group': resource_group_name, 'machine_name': vm_name}
+        # Retrieves the Arc Server object
+        return ArcServerShow(cli_ctx=cmd.cli_ctx)(command_args=get_args)
+    except Exception:
+        return None
+
+
+# This function is used to get the properties needed from a Connected VMware VM
+def _request_connected_vmware_properties(cmd, resource_group_name, vm_name):
+    from .aaz.latest.connected_v_mwarev_sphere.virtual_machine import Show as VMwarevSphereShow
+
+    # pylint: disable=broad-except
+    try:
+        get_args = {'resource_group': resource_group_name, 'machine_name': vm_name}
+        # Retrieves the Connected VMware VM object
+        return VMwarevSphereShow(cli_ctx=cmd.cli_ctx)(command_args=get_args)
+    except Exception:
+        return None
+
+
+def parse_os_type(properties, resource_type):
+    if resource_type == "microsoft.compute/virtualmachines":
+        if (properties.storage_profile and properties.storage_profile.os_disk and
+                properties.storage_profile.os_disk.os_type):
+
+            return properties.storage_profile.os_disk.os_type
+
+    elif resource_type == "microsoft.hybridcompute/machines":
+        if properties.get('osName', None):
+            return properties.get('osName')
+        if properties.get('properties', None):
+            return properties.get('properties').get('osType', None)
+
+    elif resource_type == "microsoft.connectedvmwarevsphere/virtualmachines":
+        if properties.get("osProfile") and properties.get("osProfile").get("osType"):
+            return properties.get("osProfile").get("osType")
+
+
+def parse_agent_version(properties, resource_type):
+    if resource_type == "microsoft.compute/virtualmachines":
+        return None
+
+    if resource_type == "microsoft.hybridcompute/machines":
+        if properties.get('properties'):
+            return properties.get('properties').get('agentVersion')
+
+    if resource_type == "microsoft.connectedvmwarevsphere/virtualmachines":
+        if properties.get("properties") and properties.get("properties").get('guestAgentProfile') and\
+                properties.get("properties").get('guestAgentProfile').get('agentVersion'):
+
+            return properties.get("properties").get('guestAgentProfile').get('agentVersion')
+
+
+def parse_tags(properties, resource_type):
+    if resource_type == "microsoft.compute/virtualmachines":
+        if properties.tags:
+            return properties.tags
+
+    else:
```

Comment:
I found out recently that "microsoft.connectedvmwarevsphere/virtualmachines" resource type has been deprecated, so it's probably a good idea not to add new features to it. So can you make sure here that you are only returning the tags property for "microsoft.compute/virtualmachines" and "microsoft.hybridcompute/machines"?